src/compose/utils.ts

```diff
@@ -693,3 +693,10 @@ export function dockerMountToServiceMount(
 
 	return mount as LongDefinition;
 }
+
+export function isOlderThan(currentTime: Date | null, seconds: number) {
```

Comment:
I think this should be called something like `isValidDateAndOlderThan` since it does more than just compare dates.